### PR TITLE
Initial port of colorpicker

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Resolves: blackbaud/skyux2#2084

### CODE REVIEW

File: e2e/colorpicker.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+describe('Colorpicker', () => {
+  beforeEach(() => {
+    SkyHostBrowser.get('visual/colorpicker');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+  });
+
+  it('should match previous colorpicker screenshot', (done) => {
+    expect('#screenshot-colorpicker').toMatchBaselineScreenshot(done);
+  });
```
Comment (Blackbaud-SteveBrush) [inline e2e/colorpicker.e2e-spec.ts]: Mind throwing some `xs` screenshots in here?

File: package.json [hunk 2]
```diff
@@ -25,7 +25,15 @@
     "url": "https://github.com/blackbaud/skyux-sample/issues"
   },
   "homepage": "https://github.com/blackbaud/skyux-sample#readme",
-  "peerDependencies": {},
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/core": "^3.1.0",
+    "@skyux/i18n": "^3.0.1",
+    "@skyux/popovers": "^3.0.0-rc.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: `">=3.0.0-rc.0",`

File: e2e/sample.e2e-spec.ts [hunk 3]
```diff
@@ -1,20 +0,0 @@
-import {
```
Comment (Blackbaud-SteveBrush) [inline e2e/sample.e2e-spec.ts]: Could you also delete the `./src/app/public/styles` folder?

File: package.json [hunk 4]
```diff
@@ -25,7 +25,15 @@
...
+    "@skyux/theme": "^3.1.0",
+    "rxjs": "^5.4.3"
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Your library also has a peer dependency on `@angular/forms`.

File: src/assets/locales/resources_en_US.json [hunk 5]
```diff
@@ -1,6 +1,66 @@
 {
-  "skyux_some_key": {
-    "_description": "",
-    "message": ""
+  "sky_colorpicker_alpha": {
```
Comment (Blackbaud-SteveBrush) [inline src/assets/locales/resources_en_US.json]: Should be `"skyux_"`...

File: package.json (hunk 2 above)
Comment (Blackbaud-SteveBrush) [inline package.json]: When typing `npm install` I see: 
"@skyux/popovers@3.0.0-rc.0 requires a peer of @skyux/indicators@>=3.0.0-rc.0 but none is installed. You must install peer dependencies yourself."
We should probably add indicators to this module's dev dependencies.

File: package.json [hunk 6]
```diff
@@ -25,7 +25,15 @@
     "url": "https://github.com/blackbaud/skyux-sample/issues"
```
Comment (Blackbaud-SteveBrush) [inline package.json]: The keyword "sample" is still being used in the package.json, skyuxconfig.json, and README files.

File: e2e/colorpicker.e2e-spec.ts (hunk 1 above)
Comment (blackbaud-conorwright) [reply e2e/colorpicker.e2e-spec.ts]: done

File: package.json (hunk 2 above)
Comment (blackbaud-conorwright) [reply package.json]: done

File: package.json (hunk 6 above)
Comment (blackbaud-conorwright) [reply package.json]: done

File: e2e/sample.e2e-spec.ts (hunk 3 above)
Comment (blackbaud-conorwright) [reply e2e/sample.e2e-spec.ts]: done

File: package.json (hunk 2 above)
Comment (blackbaud-conorwright) [reply package.json]: done

File: package.json (hunk 4 above)
Comment (blackbaud-conorwright) [reply package.json]: added

File: src/assets/locales/resources_en_US.json (hunk 5 above)
Comment (blackbaud-conorwright) [reply src/assets/locales/resources_en_US.json]: done

**Pull request merged**